src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
```

Comment:
If the variable name starts with the verb, it should be the function.
You got an object
```suggestion
  const inversedRobot = {};
```